R/sfi_plot_laqueur_2.R

```diff
@@ -15,163 +15,192 @@ sfi_plot_laqueur_2 <- function(){
   data <- all_data$laqueur$f2
   
   # version 1 
-  g1 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_line(size = 1, 
-              alpha = 0.6,
-              linetype = 2) +
+  g1 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_point(size = 4, 
+               alpha = 0.8,
+               color = 'black',
+               stroke = 2,
+               pch = 21) +
+    geom_line(size = 1,
+              linetype = 'dashed',
+              color = '#4D4D4D') +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%')),
+                  size = 4,
+              nudge_y = 2,
+              alpha = 0.9) +
+    theme(legend.position="none") +
+    scale_size_area(name = '',
+                    max_size = 5) +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
+         y = 'Hearings resulting in grant',
          title = 'Version 1') +
-    theme_sfi()
+    theme_sfi(lp = 'none')
   
-  # version 2 
-  g2 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 2') +
-    theme_sfi()
+
   
-  # version 3 
-  g3 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    geom_line(size = 1, 
-              alpha = 0.6) +    
+  # version 2
+  g2 <- 
```

Comment:
Do we want smoothed bands when there are so few observations? It seems a bit overkill...